app/templates/base.html

```diff
@@ -1,68 +1,79 @@
-                    <li class="nav-item">
-                        <a class="nav-link active" aria-current="page" href="{{ url_for('home') }}">Home</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link active" aria-current="page" href="{{ url_for('profile') }}">Profile</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="#">Sign In</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="#">Sign Up</a>
-                    </li>
-                    <li class="nav-item">
-                        <!-- new button for the agenda page. should be available after login
-                        place will change later according to the web design -->
-                        <a class="nav-link" href="{{ url_for('agenda') }}">Agenda</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="{{ url_for('view_invitations') }}">Invitations</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link" href="/search">Search</a> 
-                    </li>
-                </ul>
-            </div>
+    <div class="bg-gradient2 h-1"></div>
+    <div class="relative overflow-hidden">
+        <div class="relative pt-6 px-4 sm:px-6 lg:px-8">
+            <nav class="navbar navbar-expand-md navbar-light">
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggler"
+                    aria-controls="navbarToggler" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="navbar-brand">
+                    <a href="{{ url_for('home') }}" aria-label="Home">
+                        <img class="h-8 w-auto sm:h-10"
+                            src="{{ url_for('static', path='/images/icons/calendar-outline.svg') }}" alt="Logo">
+                    </a>
+                </div>
+                <div class="collapse navbar-collapse" id="navbarToggler">
+                    <ul class="navbar-nav mr-auto mb-2 mb-lg-0 ">
+                        <li class="nav-item">
+                            <a class="nav-link" href="{{ url_for('profile') }}">Profile</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="{{ url_for('not_implemented') }}">Sign In</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="{{ url_for('not_implemented') }}">Sign Up</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="{{ url_for('agenda') }}">Agenda</a>
```

Comment:
Agenda is not needs to appear in landing page.

Reply:
Thanks for the comment.
@yammesicka do you approve removing existing links?